### PR TITLE
/common/httping, /osx/ping: update page and add page

### PR DESCRIPTION
**Pull request by thiswind**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by thiswind** [comment]
The automation of this project is really high. It only needs to be pushed, and Travis is automatically built. It is really easy to use. 👍

File: pages/common/httping.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 > Measure the latency and throughput of a web server.
 
-- Ping the web server on `host`:
+- Ping the `url`, for example: `httping -g http://github.com:80/tldr-pages/tldr`:
```
Comment (pxgamer) [inline pages/common/httping.md]: The backticks aren't required for `url`. 🙂 And it's probably not necessary to add an example in the description.

Perhaps something like this?
```suggestion
- Ping the specified url:
```

But `web server` keeps it consistent with the other examples.

File: pages/osx/ping.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# ping
+
+> Send ICMP ECHO_REQUEST packets to network hosts.
+
+- Ping `host`:
```
Comment (pxgamer) [inline pages/osx/ping.md]: The backticks aren't necessary here afaik, it's only really for command syntax highlighting.

Perhaps the following would sound better?
```suggestion
- Ping the specified host:
```

File: pages/osx/ping.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- Ping `host` without trying to lookup symbolic names for addresses:
+
+`ping -n {{host}}`
+
+- Ping `host` and ring the bell when a packet is received (if your terminal supports it):
+
+`ping -a {{host}}`
+
+- Ping `host` and prints the time a packet was received.  This option is an Apple addition.:
```
Comment (pxgamer) [inline pages/osx/ping.md]:
```suggestion
- Ping `host` and prints the time a packet was received (this option is an Apple addition):
```

File: pages/osx/ping.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# ping
+
+> Send ICMP ECHO_REQUEST packets to network hosts.
+
+- Ping `host`:
+
+`ping {{host}}`
+
+- Ping a `host` only a specific number of `count`:
```
Comment (pxgamer) [inline pages/osx/ping.md]:
```suggestion
- Ping a host a specific number of times:
```

**Comment by mebeim** [comment]
Hi @thiswind, just a tip: you can check for errors in the pages before pushing using `tldr-lint`, available from `npm`:

    sudo npm install -g tldr-lint

Then you can run it on a directory or a single file to check for errors.

File: pages/common/httping.md (hunk 1 above)
Comment (thiswind) [reply pages/common/httping.md]: all right, let's take this : )

**Comment by sbrl** [comment]
Although I'm closing this, I've actually merged it. GitHub isn't detecting it for reason because I did it manually and tidied up the commits in the process. Here are the relevant commits:

 - 35949d98450303cdbfcb1a87c9bc5e876121dea1
 - 6c0eb424963181e6ec027526802aa4d7f2dc9b89

**Comment by thiswind** [comment]
thank you, I 'll make better next time :)

**Pull request closed without merging**